contracts/accountants/HealthCheckAccountant.sol

```diff
@@ -390,7 +432,7 @@ contract HealthCheckAccountant {
     function custom(
         address vault,
         address strategy
-    ) external view returns (bool) {
```

Comment:
what do you think about `isCustomConfig` or `hasCustomConfig` instead of `custom`?

Reply:
https://github.com/yearn/vault-periphery/pull/27/commits/b0aa4889148b14212c0914a3276e430c345b22ef

Went with `useCustomConfig`